pyglimmpse/multirep.py

```diff
@@ -429,16 +429,17 @@ def pbt_two_moment_null_approx_obrien_shieh(rank_C: float,
     df1, df2 = _pbt_two_moment_df1_df2(rank_C, rank_U, rank_X, total_N)
     eval_HINVE = _calc_eval(min_rank_C_U, error_sum_square, hypothesis_sum_square)
 
+    warning_message = 'Power is missing because because the min_rank_C_U - v  <= 0.'
     if _valid_df2_eigenvalues(eval_HINVE, df2, tolerance):
         evalt = _trace(eval_HINVE, rank_X, total_N)
         v = _pbt_population_value(evalt, min_rank_C_U)
         if (min_rank_C_U - v) <= tolerance:
-            warnings.warn('Power is missing because because the min_rank_C_U - v  <= 0.')
+            warnings.warn(warning_message)
```

Comment:
re add it here:
```suggestion
            warning_message = 'Power is missing because because the min_rank_C_U - v  <= 0.'
            warnings.warn(warning_message)
            return _undefined_power(warning_message)
```

Reply:
There are two possible invalids here. Thus, I will make it two different error messages.